src/Collapse.js

```diff
@@ -1,234 +1,169 @@
 import React from 'react';
-import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
 import {Motion, spring} from 'react-motion';
-import HeightReporter from 'react-height';
+import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
+
 
+const SPRING_PRECISION = 1;
 
-const PRECISION = 0.5;
 
+const WAIT = 'WAIT';
+const RESIZE = 'RESIZE';
+const REST = 'REST';
+const STABLE = 'STABLE';
 
-const stringHeight = height => Math.max(0, parseFloat(height)).toFixed(1);
 
+const noop = () => null;
+const css = {
+  collapse: 'collapse',
```

Comment:
Thanks!
 I am spoiled by CSS-modules and forgot that people still struggle with global CSS namespace